.github/workflows/build.yml

```diff
@@ -446,7 +442,7 @@ jobs:
           ls liquibase-dist/target/liquibase-${{ needs.setup.outputs.thisBranchFileName }}-SNAPSHOT
           
           ##create installer - disabled here but run as nightly job and as part of release workflow
-          # (cd liquibase-dist && ${{ github.workspace }}/.github/util/package-install4j.sh 0-SNAPSHOT)
+          (cd liquibase-dist && ${{ github.workspace }}/.github/util/package-install4j.sh 0-SNAPSHOT)
```

Comment:
Did you intend to enable this? I think it was purposefully disabled because it was time consuming and unnecessary.